### PR TITLE
Implement check_transaction to verify CreateGeneralMeeting transaction

### PR DESCRIPTION
**Pull request by MSNTCS**

### CODE REVIEW

File: voting/src/general_meeting.rs [hunk 1]
```diff
@@ -35,6 +35,7 @@ enum ExecuteError {
     TallyingTimePassed,
     TallyingIsBeforeVotingEnd,
     VotingEndPassed,
+    InvalidAdmin,
```
Comment (majecty) [inline voting/src/general_meeting.rs:38]: Actually admin is not invalid. The transaction signer is not admin.
How about `NotAuthorized` or `NotAdmin`?

File: voting/src/general_meeting.rs [hunk 2]
```diff
@@ -147,7 +148,24 @@ impl TxOwner for Context {
...
+        let todo_fixthis: coordinator::types::ErrorCode = 3;
+        match transaction.tx_type() {
+            CREATE_GENERAL_MEETING_TX_TYPE => {
+                let tx: CreateGeneralMeetingOwnTransaction =
+                    serde_cbor::from_slice(&transaction.body()).map_err(|_| todo_fixthis)?;
+                tx.verify().map_err(|_| todo_fixthis)?;
+
+                let admin_public_key = tx.signer_public;
+                let written_admin: Public =
+                    serde_cbor::from_slice(&self.storage().get(ADMIN_STATE_KEY.as_bytes()).unwrap()).unwrap();
```
Comment (majecty) [inline voting/src/general_meeting.rs:160]:
```suggestion
                let written_admin: Public = self.admin();
```

File: voting/src/general_meeting.rs [hunk 3]
```diff
@@ -147,7 +148,24 @@ impl TxOwner for Context {
...
+
+                if written_admin != admin_public_key {
+                    return Err(ExecuteError::InvalidAdmin).map_err(|_| todo_fixthis)
+                }
```
Comment (majecty) [inline voting/src/general_meeting.rs:164]: I missed this in `excute_tx`. Could you add this check in `excute_tx`?

File: voting/src/general_meeting.rs [hunk 4]
```diff
@@ -147,7 +148,24 @@ impl TxOwner for Context {
...
+                Ok(())
```
Comment (majecty) [inline voting/src/general_meeting.rs:165]: Please add this check:
```rust
                if tallying_time.time < voting_end_time.time {
                    return Err(ExecuteError::TallyingIsBeforeVotingEnd)
                }
```

Comment (MSNTCS) [reply voting/src/general_meeting.rs:177]: It is added.

**Pull request merged**